tests/helpers/in-memory-ipfs.ts

```diff
@@ -1,15 +1,26 @@
 import type { IPFS } from "ipfs-core"
 
 import * as fs from "fs"
-import * as dagPB from "@ipld/dag-pb"
 import * as Ipfs from "ipfs-core"
 import tempDir from "ipfs-utils/src/temp-dir.js"
 import { createRepo } from "ipfs-repo"
-import { BlockCodec } from "multiformats/codecs/interface"
 import { MemoryDatastore } from "datastore-core/memory"
 import { MemoryBlockstore } from "blockstore-core/memory"
 
+// Codecs
+import * as dagPB from "@ipld/dag-pb"
+import * as dagCBOR from "@ipld/dag-cbor"
+import * as raw from "multiformats/codecs/raw"
+import { BlockCodec } from "multiformats/codecs/interface"
 
+const CODECS: Record<string, BlockCodec<number, unknown>> = {
+  [dagPB.code]: dagPB,
+  [dagPB.name]: dagPB,
+  [dagCBOR.code]: dagCBOR,
+  [dagCBOR.name]: dagCBOR,
+  [raw.code]: raw,
+  [raw.name]: raw,
+}
```

Comment:
Why does this have the `.name`s as well, compared to what's in [src/fs/v3/IpfsBlockStore.ts](https://github.com/fission-codes/webnative/pull/389/files#diff-5384c7b03d3c6c7419ebd93f9a1868887231145c4484c7078f87febca960c415)? Also, I guess we should add `[raw.code]: raw` to the our new `ipfs/node/repo.ts` (which I added with my PR that just got merged)?

Reply:
The `CODECS` list in `IpfsBlockStore` is something that we manage between JS and rs-wnfs, it doesn't touch js-ipfs.
`rs-wnfs` only speaks codec codes, not codec names, so it doesn't need entries for e.g. `raw.name`.
I've taken this configuration from some issue comment that achingbrain made *somewhere*, where he's configuring a totally in-memory js-ipfs node. I can't find that comment anymore though. But it seems like js-ipfs expects there to be entries for both `codec.code` and `codec.name`.

Yep, having `raw` in there would be good!